sdk/schemaregistry/schema-registry/test/public/utils/recordedClient.ts

```diff
@@ -1,50 +1,24 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { Context } from "mocha";
-
-import { env, Recorder, record, RecorderEnvironmentSetup } from "@azure-tools/test-recorder";
-import { ClientSecretCredential } from "@azure/identity";
-
+import { createTestCredential } from "@azure-tools/test-credential";
 import { SchemaRegistryClient } from "../../../src";
+import { Recorder, RecorderStartOptions, env } from "@azure-tools/test-recorder-new";
 
-export interface RecordedClient {
-  client: SchemaRegistryClient;
-  recorder: Recorder;
-}
-
-const replaceableVariables: { [k: string]: string } = {
-  AZURE_CLIENT_ID: "azure_client_id",
-  AZURE_CLIENT_SECRET: "azure_client_secret",
-  AZURE_TENANT_ID: "azuretenantid",
-  SCHEMA_REGISTRY_ENDPOINT: "https://endpoint",
-  SCHEMA_REGISTRY_GROUP: "group-1",
-};
-
-const environmentSetup: RecorderEnvironmentSetup = {
-  replaceableVariables,
-  customizationsOnRecordings: [
-    (recording: string): string =>
-      recording.replace(/"access_token"\s?:\s?"[^"]*"/g, `"access_token":"access_token"`),
-    // If we put ENDPOINT in replaceableVariables above, it will not capture
-    // the endpoint string used with nock, which will be expanded to
-    // https://<endpoint>:443/ and therefore will not match, so we have to do
-    // this instead.
-    (recording: string): string => {
-      const replaced = recording.replace("endpoint:443", "endpoint");
-      return replaced;
-    },
-  ],
-  queryParametersToSkip: [],
+export const startOptions: RecorderStartOptions = {
```

Comment:
could `startOptions` be renamed to `recorderOptions`?

Reply:
ok